libsolidity/ast/ASTJsonImporter.cpp

```diff
@@ -341,24 +342,24 @@ ASTPointer<ContractDefinition> ASTJsonImporter::createContractDefinition(Json::V
 
 	return createASTNode<ContractDefinition>(
 		_node,
-		std::make_shared<ASTString>(_node["name"].asString()),
+		std::make_shared<ASTString>(_node["name"].get<std::string>()),
 		createNameSourceLocation(_node),
-		_node["documentation"].isNull() ? nullptr : createDocumentation(member(_node, "documentation")),
+		_node.contains("documentation") ? (_node["documentation"].is_null() ? nullptr : createDocumentation(member(_node, "documentation"))) : nullptr,
```

Comment:
```suggestion
		(_node.contains("documentation") && !_node["documentation"].is_null()) ? createDocumentation(member(_node, "documentation")) : nullptr,
```
Not that it matter much, but this may be slightly less pathological, won't id?

Reply:
True! I wanted to fix this once it is working correctly, but then I forgot.